### PR TITLE
[action] remove nightly from update_fastlane

### PR DESCRIPTION
**Pull request by joshdholtz**
### Motivation and Context
Remove nightly builds because the bot that uploaded them was not able to have 2FA enabled which made the bot susceptible for take over 😱

### Description
- Deprecated option for `nightly`
- Also removed deprecated `tools` action as super no longer relevant

### CODE REVIEW

File: fastlane/lib/fastlane/actions/update_fastlane.rb [hunk 1]
```diff
@@ -134,21 +98,17 @@ def self.details
...
-                                       env_name: "FL_UPDATE_FASTLANE_NIGHTLY",
-                                       description: "Opt-in to install and use nightly fastlane builds",
-                                       is_string: false,
-                                       default_value: false),
           FastlaneCore::ConfigItem.new(key: :no_update,
                                        env_name: "FL_NO_UPDATE",
                                        description: "Don't update during this run. This is used internally",
                                        is_string: false,
                                        default_value: false),
-          FastlaneCore::ConfigItem.new(key: :tools,
```
Comment (janpio) [inline fastlane/lib/fastlane/actions/update_fastlane.rb]: Was this intentional that this is also removed in this PR?

File: fastlane/lib/fastlane/actions/update_fastlane.rb [hunk 2]
```diff
@@ -134,21 +98,17 @@ def self.details
...
-                                       env_name: "FL_TOOLS_TO_UPDATE",
-                                       description: "Comma separated list of fastlane tools to update (e.g. `fastlane,deliver,sigh`)",
-                                       deprecated: true,
-                                       optional: true)
```
Comment (janpio) [inline fastlane/lib/fastlane/actions/update_fastlane.rb]: Was this intentionally removed in this PR?

Comment (joshdholtz) [reply fastlane/lib/fastlane/actions/update_fastlane.rb]: Yup yup! It is sooooo old. This is when _fastlane_ had multiple gems (one for each tool). I was triggered when I saw it so I wanted it gone too while I was in here 🙃

Comment (janpio) [reply fastlane/lib/fastlane/actions/update_fastlane.rb]: Bye bye then

**Review by janpio — APPROVED** [review]

**Pull request merged**